android/engine/src/main/java/org/smartregister/fhircore/engine/domain/model/ProfileData.kt

```diff
@@ -82,11 +82,14 @@ sealed class ProfileData(open val logicalId: String, open val name: String) {
     override val name: String,
     val identifier: String? = null,
     val birthdate: Date,
-    val age: String,
+    val age: String = birthdate.toAgeDisplay(),
     val gender: Enumerations.AdministrativeGender,
     val address: String,
     val phoneContacts: List<ContactPoint> = listOf(),
-    val chwAssigned: Reference
+    val chwAssigned: Reference,
+    val patientType: PatientType,
+    val services: List<CarePlan> = listOf(),
```

Comment:
it makes sense that we'd need this, how are we handling this now though?

Reply:
Hi @pld , its is not in use for now, as it was requested and discussed before that patient profile detail would be require with field of assigned-chw, this property is added there, further @KhumboLihonga can better comment how they plan to use chw in profile data.

Reply:
Care plans are supposed to be use for different clinic visit of a patient, 
@Gental-Giant can comment on this